samples/msgext-northwind-inventory-csharp/lab/Exercise 01 - Set up.md

```diff
@@ -0,0 +1,95 @@
+
+2. [Azure Storage Explorer](https://azure.microsoft.com/products/storage/storage-explorer/) (OPTIONAL) - Download this if you want to view and edit the Northwind database used in this sample
+
+## Step 2 - Download the sample code
+
+Please [clone](https://github.com/OfficeDev/Copilot-for-M365-Plugins-Samples.git) or [download](https://github.com/OfficeDev/Copilot-for-M365-Plugins-Samples.git) the sample repository: [https://github.com/OfficeDev/Copilot-for-M365-Plugins-Samples/](https://github.com/OfficeDev/Copilot-for-M365-Plugins-Samples/).
+
+Within the cloned or downloaded repository, navigate to the **samples/msgext-northwind-inventory-csharp** folder. These labs will refer to this as your "working folder" since this is where you'll be working.
+
+## Step 3 - Copy sample documents to your test user's OneDrive
+
+The sample application includes some documents for Copilot to reference during the labs. In this step you will copy these files to your user's OneDrive so Copilot can find them. Depending on how the tenant is set up, you may be asked to set up multi-factor authentication as part of this process.
+
+Open your browser and browse to Microsoft 365 ([https://www.office.com/](https://www.office.com/)). Log in using the Microsoft 365 account you will be using throughout the lab. You may be asked to set up multi-factor authentication.
+
+Using the "waffle" menu in the upper left corner of the page 1️⃣ , navigate to the OneDrive application within Microsoft 365 2️⃣ .
+
+![Navigating to the OneDrive application in Microsoft 365](./images/01-02-CopySampleFiles-01.png)
+
+Within OneDrive, navigate to "My Files" 1️⃣ . If there's a documents folder, click into that as well. If not, you can work directly within the "My Files" location.
+
+![Navigating to your documents in OneDrive](./images/01-02-CopySampleFiles-02.png)
+
+Now click "Add new" 1️⃣ and "Folder" 2️⃣ to create a new folder.
+
+![Adding a new folder in OneDrive](./images/01-02-CopySampleFiles-03.png)
+
+Name the folder "Northwind contracts" and click "Create".
+
+![Naming the new folder "Northwind contracts"](./images/01-02-CopySampleFiles-03b.png)
+
+Now, from within this new folder, click "Add new" 1️⃣  again but this time click "Files upload" 2️⃣ .
+
+![Adding new files to the new folder](./images/01-02-CopySampleFiles-04.png)
+
+Now browse to the **sampleDocs** folder within your working folder. Highlight all the files 1️⃣ and click "OK" 2️⃣  to upload them all.
+
+![Uploading the sample files from this repo into the folder](./images/01-02-CopySampleFiles-05.png)
+
+By doing this step early, there's a good chance that the Microsoft 365 search engine will have discovered them by the time you're ready for them.
+
+## Step 4 - Install and set up Teams Toolkit for Visual Studio Code
+
+Install Teams Toolkit for Visual Studio [Teams Toolkit extension](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/toolkit-v4/install-teams-toolkit-vs?pivots=visual-studio-v17-7)
```

Comment:
Teams Toolkit for Visual Studio is a component in the ASP.NET workload so we might want to remove the word `extension` here as that is something different in Visual Studio.

Reply:
fixed